tests/atomizer.js

```diff
@@ -307,6 +307,53 @@ describe('getConfig()', function () {
         config = atomizer.getConfig(classNames, defaultConfig);
         expect(config).to.deep.equal(atomizer.mergeConfigs([defaultConfig, expectedConfig]));
     });
+
+    it("should append the default unit", function () {
+        var classNames = ['P-1', 'M-0.3', 'P-2%', 'C-333'];
+
+        var defaultConfig = {
+            defaultUnit: 'px',
+            margin: {
+                defaultUnit: 'em'
```

Comment:
@src-code @thierryk I though about that same issue too, and as you can see it is possible to have different default units for different properties. In this tests you can see that the defaultUnit set on the top level is `px` but we are able to override it to use `em` for margin.

This setting will be very helpful to the finance project not only for the migration but also to make sure we have a consistent use units, I do not want to allow people to use `px` in some paddings and `em` in others I want all paddings on the app to use the same unit to keep the ui consistent.